Cargo.toml

```diff
@@ -89,6 +89,3 @@ travis-ci = { repository = "paritytech/polkadot", branch = "master" }
 maintenance = { status = "actively-developed" }
 is-it-maintained-issue-resolution = { repository = "paritytech/polkadot" }
 is-it-maintained-open-issues = { repository = "paritytech/polkadot" }
-
-[profile.release]
-panic = "abort"
```

Comment:
Even though `panic=unwind` is the default, I'd rather specify it explicitly with a comment for documentation purposes.